extensions/common/iam/identity-trust/identity-trust-issuers-configuration/README.md

```diff
@@ -0,0 +1,13 @@
+# Trusted Issuer Configuration Extension
```

Comment:
Can we get rid of this documentation and just use the annotations for this?

Reply:
Do we support the prefix style in annotations?